features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the previous and current branches are deleted during a Git Town command, the main branch becomes the new previous branch
+
+  (see ../same_current_branch/previous_branch_same.feature)
+
+
+  Scenario: git-prune-branches
+    Given I have branches named "previous" and "current"
+    And I am on the "current" branch with "previous" as the previous Git branch
+    When I run `git prune-branches`
+    Then I end up on the "main" branch
+    And my previous Git branch is now "main"
```

Comment:
I treat `main` as a default branch. If I'm unable to find the true previous branch, `main` is used.

I think it's possible to find the previous branch that isn't main, but that adds complexity and I don't think yields much benefit. Do you as a user even know what branch that would be? I don't think users tend to keep track of their branch history past the `n-1` branch.